### PR TITLE
feat: add relay dev tools on flipper

### PR DESCRIPTION
**Pull request by gkartalis**
This PR resolves [] 

### Description

Adds relay-dev-tools in flipper so we can debug what's going on with the relay store 🤩

Screenshot 2023-03-21 at 10 48 37

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- add relay dev tools on flipper - gkartalis

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Comment by gkartalis** [comment]
if someone can pull down the branch and see if flipper is getting automatically included in flipper it would be great.

If not I will also update the docs to reflect the [installation](https://github.com/th3rdwave/flipper-plugin-relay-devtools#flipper-desktop-app) of the plugin

File: src/app/App.tsx [hunk 1]
```diff
@@ -36,9 +36,11 @@ import { usePreferredThemeTracking } from "./utils/usePreferredThemeTracking"
 import { useScreenReaderTracking } from "./utils/useScreenReaderTracking"
 import useSyncNativeAuthState from "./utils/useSyncAuthState"
 
-// Don't open RN dev menu with shake. We use it for our own Dev Menu.
 if (__DEV__) {
+  // Don't open RN dev menu with shake. We use it for our own Dev Menu.
```
Comment (dimatretyak) [inline src/app/App.tsx:40]: 🙏

File: src/app/App.tsx [hunk 2]
```diff
@@ -36,9 +36,11 @@ import { usePreferredThemeTracking } from "./utils/usePreferredThemeTracking"
...
   NativeModules.DevSettings.setIsShakeToShowDevMenuEnabled(false)
+  // Add flipper relay devtools only in dev mode
+  require("react-native-flipper-relay-devtools").addPlugin()
```
Comment (dimatretyak) [inline src/app/App.tsx:43]: Nice. Very useful devtool 😎

**Review by dimatretyak — APPROVED** [review]
Thank you!

**Comment by MounirDhahri** [comment]
@gkartalis Forgot to comment - this works for me 🙌 thanks for the addition

**Pull request merged**